actions/create_archive_dir.yaml

```diff
@@ -0,0 +1,43 @@
+---
+    name: 'create_archive_dir'
+    runner_type: 'run-local'
+    description: 'Creates a copy of the runfolder, consisting of symlinks, for archiving purposes.'
+    enabled: true
+    entry_point: ''
+    parameters:
+        timeout:
+            default: 86400
+        runfolder:
+            type: 'string'
+            description: 'Path to runfolder to archive'
+            required: true
+        fastq_threshold:
+            type: 'integer'
+            description: 'Action will fail if less than this number of fastq files are found'
+            required: true
+        remove_previous: 
+            type: 'boolean'
+            description: 'Set to true if we want to remove an already existing archive directory'
+            required: true
+            default: false
+        host:
+            type: 'string'
+            description: 'Host where runfolder is located'
+            required: true
+        python_path:
+            type: 'string'
+            description: 'Path to the Python 2.7 binary to use'
+            required: true
+            default: '/opt/arteria/arteria-runfolder-env/bin/python'
+        exclude_links: 
+            type: 'string'
+            description: 'Argument to script for which links we should not create'
+            required: false
+            default: '-e Data -e Thumbnail_Images'
+        cmd:
+            immutable: true 
+            default: 'ssh -v {{host}} "{{python_path}} -u - {{exclude_links}} -t {{fastq_threshold}} --remove={{remove_previous}} {{runfolder}}" < /opt/stackstorm/packs/arteria-packs/actions/lib/create_archive_dir.py'
```

Comment:
Cool. Do I understand correctly that you are passing the python script on the master node to be run on the `host` node? Looks like a very useful construct that we might want to wrap up and reuse later. :+1: